apps/src/code-studio/pd/workshop_enrollment/WorkshopJoin.tsx

```diff
@@ -40,31 +41,62 @@ const WorkshopJoin: React.FunctionComponent<{
     session_info_for_calendar?: SessionCalendarEvent[];
   };
   user_info: {
+    id: number;
     display_name: string;
     given_name?: string;
     family_name?: string;
     email: string;
-    school_name?: string;
+    school_info?: {
+      school_id?: number;
+      country?: string;
+      school_name?: string;
+      zip?: string;
+      school_type?: string;
+      school_state?: string;
+      school_district_name?: string;
+      school_district_other?: string;
+    };
```

Comment:
This info will be cleaned up very soon when we simplify the enrollment process to just take the user id (then we extrapolate all other info from that).